docs/usecases/datastorage_for_institutions.rst

```diff
@@ -0,0 +1,314 @@
+Git-annex remote for indexed file archives (`RIA <https://libraries.io/pypi/ria-remote>`_)
+special remote. The Git-annex RIA-remote is similar to Git-annex's built-in
+`directory <https://git-annex.branchable.com/special_remotes/directory/>`_
+special remote, but distinct in certain aspects:
+
+- It allows read-access to (compressed) 7z archives, which is a useful
+  feature on systems where strong quotas on filesystem inodes might be imposed
+  on users, or where one wants to have compression gains.
+  This way, the entire key store (i.e., all data contents) of the
+  remote that serves as the data store can be put into an archive that uses
+  only a handful of inodes, while remaining fully accessible.
+
+- It provides access to configurable directories via SSH.
+
+  .. todo::
+
+     understand why this is special/cool.
+
+- It allows a multi-repository directory structure, in which key store
+  directories of multiple repositories can be organized in to a homogeneous
+  archive directory structure. Importantly, the key store location in an archive is defined
+  using the **datasets UUID** (in case of DataLad datasets) or the annex remote
+  UUID (in case of any non DataLad dataset). This aids handling of large
+  numbers of repositories in a data store use case, because locations are
+  derived from *repository properties* rather than having to re-configure them explicitly.
+
+The structure under which data is stored in the data store looks like this:
+
+.. code-block::
+   :emphasize-lines: 1-2, 4-10
+
+    082
+    ├── 8ac72-f7c8-11e9-917f-a81e84238a11
+    │   ├── annex
+    │   │   ├── objects
+    │   │   │   ├── ff4
+    │   │   │   │   └── c57
+    │   │   │   │       └── MD5E-s4--ba1f2511fc30423bdbb183fe33f3dd0f
+    │   │   │   ├── abc
+    │   │   │   │   └── def
+    │   │   │   │       └── MD5E-s4--ba1f2511fc30423bdbb183fe33f3dd0f
+    │   │   │   ├── [...]
+    │   │   └── archives
+    │   │       └── archive.7z
+    │   ├── branches
+    │   ├── config
+    │   ├── description
+    │   ├── HEAD
+    │   ├── hooks
+    │   │   ├── [...]
+    │   ├── info
+    │   │   └── exclude
+    │   ├── objects
+    │   │   ├── 04
+    │   │   │   └── 49b485d128818ff039b4fa88ef57be0cb5b184
+    │   │   ├── 06
+    │   │   │   └── 4e5deab57592a54e4e9a495cde70cd6da7605a
+    │   │   ├── [...]
+    │   │   ├── info
+    │   │   └── pack
+    │   ├── refs
+    │   │   ├── heads
+    │   │   │   ├── git-annex
+    │   │   │   └── master
+    │   │   └── tags
+    │   └── ria-layout-version
+    └── c9d36-f733-11e9-917f-a81e84238a11
+        ├── [...]
+
+Here is how the RIA-remote features look like in real life:
+
+- Datasets are identified via their :term:`UUID` (e.g.,
+  ``0828ac72-f7c8-11e9-917f-a81e84238a11``). The UUID is split into the first
+  two levels of the tree structure (as highlighted above in the first two
+  lines), with the two-level structure preventing too many inodes underneath
+  a single directory.
```

Comment:
"with the two-level structure to avoid exhausting file system limits on the number of files/folders within a directory"